src/items/class-features/devastating_throw_(ex).json

```diff
@@ -0,0 +1,139 @@
+  "_id": "D7CQIofs6gzzkomY",
+  "name": "Devastating Throw (Ex)",
+  "type": "feat",
+  "_stats": {
+    "coreVersion": "10.288",
+    "createdTime": 1668401270501,
+    "lastModifiedBy": "SPRszH5sDdJlGxJn",
+    "modifiedTime": 1668401967797,
+    "systemId": "sfrpg",
+    "systemVersion": "0.21.0"
+  },
+  "img": "icons/svg/mystery-man.svg",
+  "system": {
+    "type": "",
+    "ability": null,
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "",
+    "activation": {
+      "type": "",
+      "condition": "",
+      "cost": 0
+    },
+    "area": {
+      "effect": "",
+      "shape": "",
+      "units": "",
+      "value": 0
+    },
+    "attackBonus": 0,
+    "chatFlavor": "",
+    "critical": {
+      "effect": "",
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.wFTC0sGqdpnZUib4]{Gear Boost}</p>\n<p><span class=\"fontstyle0\">You hurl thrown weapons with frightful strength. You treat the range increments of thrown weapons as 10 feet greater than normal. If your Strength modifier is higher than your Dexterity modifier, whenever you throw a weapon and add your Dexterity modifier to the attack roll, you gain a +1 bonus to the attack roll. If your Strength modifier is at least 3 higher than your Dexterity modifier, you instead gain a +2 bonus to the attack roll. <br></span></p>\n<p><em><span class=\"fontstyle0\"><br>Edit modifier in the Modifiers tab.</span></em></p>"
+    },
+    "descriptors": {
+      "acid": false,
+      "air": false,
+      "calling": false,
+      "chaotic": false,
+      "charm": false,
+      "cold": false,
+      "compulsion": false,
+      "creation": false,
+      "curse": false,
+      "darkness": false,
+      "death": false,
+      "disease": false,
+      "earth": false,
+      "electricity": false,
+      "emotion": false,
+      "evil": false,
+      "fear": false,
+      "fire": false,
+      "force": false,
+      "good": false,
+      "healing": false,
+      "language-dependent": false,
+      "lawful": false,
+      "light": false,
+      "mind-affecting": false,
+      "pain": false,
+      "poison": false,
+      "polymorph": false,
+      "radiation": false,
+      "scrying": false,
+      "sense-dependent": false,
+      "shadow": false,
+      "sonic": false,
+      "summoning": false,
+      "teleportation": false,
+      "water": false
+    },
+    "duration": {
+      "units": "",
+      "value": null
+    },
+    "formula": "",
+    "isActive": null,
+    "modifiers": [
+      {
+        "_id": "f8e6569c-b7da-435b-ab0d-722677109966",
+        "name": "Devastating Throw (Ex)",
+        "type": "untyped",
+        "condition": "",
+        "effectType": "all-attacks",
+        "enabled": false,
+        "max": 1,
+        "modifier": "1",
```

Comment:
This may be fully automatable, if you calculate the difference between str and dex, and use that in a lookupRange